lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -165,6 +187,22 @@ def restart_evmserverd
         evmserverd_service = LinuxAdmin::Service.new("evmserverd")
         evmserverd_service.restart if evmserverd_service.running?
       end
+
+      def deactivate
+        @secure = nil
+        @server_host_is_ipaddr = nil
+        configure_messaging_type("miq_queue") # Settings.prototype.messaging_type = 'miq_queue'
+        restart_evmserverd
+        remove_installed_files
+      end
+
+      def secure?
+        @secure ||= server_port == 9_093
+      end
+
+      def server_host_is_ipaddr?
+        @server_host_is_ipaddr ||= server_host =~ Prompts::IP_REGEXP
```

Comment:
Can this use `String#ipaddress?` from more_core_extensions instead?

Reply:
Thank you. I will explore that.

Reply:
Addressed by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/149/commits/1438dd4126fec5988899d13c39f018c70e1c641c